test/setup.js

```diff
@@ -0,0 +1,75 @@
+"use strict";
+
+const {createInstrumenter} = require("istanbul-lib-instrument");
+const fs = require("fs");
+const {jsdom, createVirtualConsole} = require("jsdom");
+
+var virtualConsole = createVirtualConsole().sendTo(console);
+
+// Path to the browser-polyfill script, relative to the current work dir
+// where mocha is executed.
+const BROWSER_POLYFILL_PATH = "./dist/browser-polyfill.js";
+
+function setupTestDOMWindow(chromeObject) {
+  return new Promise((resolve, reject) => {
+    let window;
+
+    // If a jsdom window has already been created, reuse it so that
+    // we can retrieve the final code coverage data, which has been
+    // collected in the jsdom window (where the instrumented browser-polyfill
+    // is running).
+    if (global.window) {
```

Comment:
ah, yeah! that's true, I removed any `global` usage in the last updates pushed in this pull request